tests/functional/SprykerEco/Zed/Computop/Business/Api/AbstractPaymentTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Functional\SprykerEco\Zed\Computop\Business\Api;
+
+use GuzzleHttp\Psr7;
+use Orm\Zed\Computop\Persistence\SpyPaymentComputop;
+use Orm\Zed\Computop\Persistence\SpyPaymentComputopDetail;
+use Orm\Zed\Computop\Persistence\SpyPaymentComputopOrderItem;
+use SprykerEco\Service\Computop\ComputopService;
+use SprykerEco\Shared\Computop\ComputopConstants;
+use SprykerEco\Zed\Computop\Business\ComputopBusinessFactory;
+use SprykerEco\Zed\Computop\Dependency\Facade\ComputopToComputopServiceBridge;
+use SprykerEco\Zed\Computop\Persistence\ComputopQueryContainer;
+
+abstract class AbstractPaymentTest extends AbstractFactoryPaymentTest
```

Comment:
PLease try to avoid AbstractTest classes that makes the tests harder to understand. MOve code you need in more than one test class to Helper